pkg/trace/config/apply.go

```diff
@@ -382,6 +382,11 @@ func (c *AgentConfig) applyDatadogConfig() error {
 		c.LogThrottling = false
 	}
 
+	// undocumented
+	if config.Datadog.IsSet("apm_config.enable_remote_configuration_dynamic_rates") {
+		c.RemoteRates = config.Datadog.GetBool("apm_config.enable_remote_configuration_dynamic_rates")
```

Comment:
This is too crazy and long. Can't we shorten it? e.g. `apm_config.remote_rates: true`

Note that you also have to add these to `pkg/config/apm.go`